var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -40,7 +37,12 @@ class Pfunit(CMakePackage):
 
     depends_on('python@2.7:', type=('build', 'run'))  # python3 too!
     depends_on('mpi', when='+mpi')
+    depends_on('m4', when='@4.1.5:')
 
+    conflicts("%gcc@:8.3.9", when="@4.0.0:", msg='Older versions of GCC do '
+              'not support the Fortran 2008 features required by new pFUnit.')
+    # See https://github.com/Goddard-Fortran-Ecosystem/pFUnit/pull/179
```

Comment:
Does the comment lin line 44 (PR 179) go with the "gcc@8.3.9" conflicts?  If so, it should be moved just above the "conflicts" statement.

Reply:
Nope, it goes with the shared conflict.